src/pages/Cart/Cart.jsx

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import "./Cart.scss";
+import QuantityCounter from "../../components/QuantityCounter/QuantityCounter";
+import { products } from "../../fakeData";
+
+const Cart = () => {
+    return (
+        <main className="main container">
+            <section className="contents">
+                <div className="title">Items in Cart: {products.length}</div>
+
+                {products.map((product) => (
+                    <div className="card-cart" key={product.id}>
+                        <img src={product.img} alt={product.title} />
+                        <div className="card-details">
+                            <div className="heading">
+                                <h4>
+                                    {product.details}{" "}
+                                    <span>{`₹${product.price}`}</span>
+                                </h4>
+                                <h6>{product.title}</h6>
+                            </div>
+
+                            <p>Black/Rough Green/White/Total Orange</p>
+
+                            <div className="order__details">
+                                <div className="size">
+                                    <label htmlFor="size">Size</label>
+                                    <select
+                                        name="variant"
+                                        id="variant"
+                                        className="variant"
+                                    >
+                                        <option>6</option>
+                                        <option>7</option>
+                                        <option>8</option>
+                                        <option>9</option>
+                                        <option>10</option>
+                                        <option>11</option>
+                                    </select>
+                                </div>
+
+                                <div className="quantity">
+                                    <label htmlFor="quantity__counter">
+                                        Quantity
+                                    </label>
+                                    <QuantityCounter />
+                                </div>
+                            </div>
+
+                            <div className="order__actions">
+                                <button className="btn">
+                                    Move to Favourites
+                                </button>
+                                <button className="btn">Remove</button>
+                            </div>
+                        </div>
+                    </div>
+                ))}
+            </section>
+
+            <section className="order__summary">
+                <div className="title">Order Summary</div>
+
+                <ul className="summary">
+                    <li>
+                        Subtotal <span>₹ 28,185</span>
```

Comment:
change price according to the item in the cart, currently it is only showing static price

Reply:
Yes, I did this just to get a view of what my final page will look like. I will be refactoring it when I will add API for backend.